documentation/contributing/contributor-guide.adoc

```diff
@@ -0,0 +1,86 @@
+[[contributing-to-docs]]
+= Contributing to the Documentation
+
+The following guide walks you through contributing to the Strimzi documentation.
+
+[[contribution-getting-started]]
+== Getting Started
+
+Before you begin you need to:
+
+. xref:setting-up-git[Install and set up Git]
+
+[[contributing]]
+== Contributing to the Documentation
+
+Once you have your local repository set up and have up to date copies of upstream content you need to follow these steps to contribute to the documentation.
+
+.Procedure
+. Open your terminal
+. `cd` to the directory where your documentation resides
+. Checkout the master branch
++
+[source]
+----
+$ git checkout master
+----
+. Update your local repository and fork with the upstream content
++
+[source]
+----
+$ git pull upstream master
+$ git push origin master --force
+----
+. Create a new branch for your work (using the issue number is helpful)
++
+[source,options="nowrap",subs="+quotes"]
+----
+$ git checkout -b <branch-name>
+----
+. Make your edits in the editor of your choice
+. Save your changes locally
+. If you are creating new files, add the files to the repository
++
+[source,options="nowrap",subs="+quotes"]
+----
+$ git status
+$ git add <file-name>
+----
+
+. Commit your changes
++
+[source,options="nowrap",subs="+quotes"]
+----
+$ git commit -a -m "<message>"
+----
+. Push your changes to your fork
++
+[source]
+----
+$ git push origin HEAD
+----
+. If the update is rejected because the commit is behind, merge your changes
++
+[source]
+----
+$ git pull upstream master
+$ git push -f origin HEAD
+----
+. Visit your fork on GitHub
+. Click *Compare & pull request*
+. If a SME review is needed:
```

Comment:
Can we define SME, and how would a contributor know whether a SME review was needed?

Reply:
Yes, let's enter an issue for me to add more info to this section.  To give you a quick answer - a SME Review is a Subject Matter Expert and a review by a SME is almost always needed. The only time you wouldn't need one is if you were fixing a broken link or a typo.

Reply:
I raised #607. Are GitHub issues enough for this? Or do we need JIRAs?